### ISSUE TITLE
Fitting range in batch mode cannot be set for each dataset

### ISSUE THREAD

**Issue by drnecj** [SasView/sasview#1819]
Hello,
when I try to use the batch mode I cannot change the fitting range. It always fall back to the default range after I click to set it for different dataset.

**Comment by drnecj** [comment]
I'm not sure what version it is exactly....
![Screenshot from 2021-04-19 20-19-34](https://user-images.githubusercontent.com/11394150/115284541-19a92000-a14d-11eb-9a71-62dca3c1e8a4.png)

**Comment by smk78** [comment]
@drnecj , you are correct that if you change the selected dataset in the drop-down on the BatchPage that the Q limits reset, but in order to fit all your selected data to the same Q limits this is unnecessary. Just select one dataset, change the limits and click fit. All the selected data will then be fit to those limits as you can see here:
![image](https://user-images.githubusercontent.com/10938679/115302173-0b153580-a15a-11eb-9348-058df02b180f.png)
(ignore the accuracy of the fits; this was just a quick test!)

**Comment by butlerpd** [comment]
Question is whether there is a suggested change to make.  At the least it may be useful to figure out how to minimize confusion on what you can do by clicking through the data sets?  Another option might be to allow each dataset to have a different set of options (polydispersity, q limits, use dq or not, use dI or not etc etc).  on the face of it this sounds kinda reasonable but the more I think about it the more this seems like total madness.
